lib/workers/repository/update/branch/index.ts

```diff
@@ -114,19 +126,37 @@ export async function processBranch(
   const artifactErrorTopic = emojify(':warning: Artifact update problem');
   try {
     // Check if branch already existed
-    const existingPr = branchPr ? undefined : await prAlreadyExisted(config);
+    const existingPr = await prAlreadyExisted(config);
     if (existingPr && !dependencyDashboardCheck) {
-      logger.debug(
-        { prTitle: config.prTitle },
-        'Closed PR already exists. Skipping branch.'
-      );
-      await handlepr(config, existingPr);
-      return {
-        branchExists: false,
-        prNo: existingPr.number,
-        result: BranchResult.AlreadyExisted,
-      };
+      // Recreates pr if merged pr already exists
+      if (existingPr.state === 'merged') {
+        if (config.automerge) {
+          logger.debug(
+            'Merged pr with the same title already exists, disable automerge'
+          );
+        }
+
+        config = { ...config, ...recreateMergedPrConfig(branchPr) };
+        if (config.recreateMergedPr) {
+          logger.debug(
+            { prTitle: config.prTitle },
+            'Merged PR already exists. Creating new PR with automerge disabled'
```

Comment:
Won't this result in two very similar debug messages being logged in a row? i.e. this and line 135